grading/README.md

```diff
@@ -0,0 +1,12 @@
+# 452 Grading
+
+## Files
+- config.json: contains the test configuration (what tests to run, where they are, where the student submissions are, what servers to use, where the jars/etc are). There are some old configuration parameters for git integration, but I didn't finish that up.
+- distributor.py: splits up the students into groups for each of the hosts that the tests will be run on. Rsyncs the students to grade onto each host and starts to run grader.py.
+- grader.py: grades on a local machine with many command line parameters corresponding to all of the required items.
+- test-resources: These are the test resources needed to compile: things like the framework and test jars, also the Makefile that run-tests.py uses.
+- Grades: Where I will move the grades after I invoke the grader.
+- Grade Exporter: A script that can import the grades into catalyst. Look at the [readme](grade_exporter/README.md) for more info.
```

Comment:
This no longer exists, right?

Reply:
Yeah, let me take that out and perhaps add to the README a bit more.